metaflow/parameters.py

```diff
@@ -331,6 +331,12 @@ def __init__(
                 "for string parameters." % name
             )
 
+    def __repr__(self):
+        return f"metaflow.Parameter(name={self.name}, " f"kwargs={self.kwargs})"
```

Comment:
we don't use f strings much since we haven't yet officially deprecated python 3.5 for core metaflow. our philosophy for core functionality is that we actively try not to break version compatibility for existing functionality.